src/components/config/parser/insomnia.ts

```diff
@@ -110,10 +109,14 @@ export function parseInsomnia(configString: string, format: string): Config {
     ? `${env?.scheme?.[0] ?? 'http'}://${env?.host}${env?.base_path}`
     : ''
 
-  return mapInsomniaToConfig(data)
+  return mapInsomniaToConfig(data, environmentVariables, baseUrl)
 }
 
-function mapInsomniaToConfig(data: unknown): Config {
+function mapInsomniaToConfig(
```

Comment:
for effectivity mapInsomniaToConfig and mapInsomniaRequestToConfig can be merged. By making mapInsomniaToConfig as a pure function that return insomniaRequests array, then call it from inside mapInsomniaRequestToConfig rather than just passing parameters to the next function.